Kernel/FileSystem/DevPtsFS.cpp

```diff
@@ -12,9 +12,10 @@
 
 namespace Kernel {
 
-ErrorOr<NonnullRefPtr<DevPtsFS>> DevPtsFS::try_create()
+ErrorOr<NonnullRefPtr<FileSystem>> DevPtsFS::try_create()
 {
-    return adopt_nonnull_ref_or_enomem(new (nothrow) DevPtsFS);
+    auto fs = TRY(adopt_nonnull_ref_or_enomem(new (nothrow) DevPtsFS));
+    return fs;
```

Comment:
No need for the temporary variable:

```suggestion
    return TRY(adopt_nonnull_ref_or_enomem(new (nothrow) DevPtsFS));
```

Same everywhere below